tools/CouchDB_Confirm_Integrity.php

```diff
@@ -80,31 +81,40 @@ function run()
                     "VL" => $vl
                 )
             );
+            $inactiveCandID = $this->SQLDB->pselectRow("SELECT * FROM candidate c WHERE c.Active='N' AND c.PSCID=:PID", array("PID" => $pscid));
```

Comment:
I don't see why this extra query is required over just adding `Active='Y'` (for both the session and candidate) to the query above just above it (that isn't showing up in the diff.)